src/main.ts

```diff
@@ -35,6 +37,18 @@ const bootstrap = async () => {
     })
   );
 
+  app.use(
+    session({
+      secret: 'keyboard cat',
```

Comment:
what is this for?

Reply:
Leftover - session key...

Reply:
It is a secret to sign the sessions cookie, should be changed to something more secure like configurable hash. Have to be preserved.

Reply:
Fixed in ba4381f9fc391608ca423d5692abd6b9ed64c4fe